code/_onclick/click.dm

```diff
@@ -270,7 +270,7 @@
 	return
 
 /atom/proc/AltClick(var/mob/user)
-	if(!(user == src) && !(isrobot(user)) && ishuman(src) && user.Adjacent(src))
+	if(!(user == src) && !(isrobot(user)) && iscarbon(src) && user.Adjacent(src))
 		src:give_item(user)
```

Comment:
I realize you didn't write this, but while you're touching the file, I'd appreciate it if you would replace `:` with `.`.
`:` is nearly always unsafe because it allows a proc to be called without checking that the datum's type has that proc beforehand, meaning a lot of stuff breaks if it turns out to not have the proc.